backend/endpoints/threads.py

```diff
@@ -168,42 +184,33 @@ async def update_threads_message(
     org_id: str,
     room_id: str,
     message_id: str,
+    thread_id: str,
```

Comment:
why not provide this as a path parameter for the purpose of consistence?

Reply:
Yes please, the id of the resource to be edited must be provided in the route. @Shopiley

Reply:
Also please add a docstring to the function.